app/components/admin/editions/document_history_tab_component.html.erb

```diff
@@ -0,0 +1,47 @@
+<h2 class="govuk-heading-l">History</h2>
+
+<% if editing %>
+  <p class="govuk-body">To add an internal note, save your changes.</p>
+<% end %>
+
+<p class="govuk-body"><%= link_to("Add internal note", new_admin_edition_editorial_remark_path(edition), class: "govuk-body govuk-link") %></p>
```

Comment:
Is the "Add internal note" button supposed to be hidden if editing?

Reply:
It should show. The copies just a prompt to save as it takes you off the page. Maybe the copy should be.

```
Before adding an internal note, save your changes.
```